spec/fixtures/contacts.yml

```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
These are identifiers so you can access your fixtures in your tests, i.e. `contacts(:one)`. The ones created by the generator are meant to be examples and you should replace them with more meaningful names, e.g:
```yml
jane_doe:
  id: 1
  user_id: 1
  name: Jane Doe
  birthday: 2009-03-15
```
And use it as `contacts(:jane_doe)` in your specs.